### ISSUE TITLE
IBaubleRender.Helper.rotateIfSneaking has incorrect positioning

### ISSUE THREAD

**Issue by yrsegal** [williewillus/Botania#237]
`GlStateManager.translate(0.0, 0.3, 0.0)` works at no scale changing with rendertype `HEAD`

with rendertype `BODY`, `GlStateManager.translate(0.0, 0.2, 0.0)`

**Comment by yrsegal** [comment]
Just realized this dupes #101, but it provides a solution to it

**Comment by williewillus** [comment]
Wait idk what your message is saying to change :p

**Comment by yrsegal** [comment]
`IBaubleRender.Helper.rotateIfSneaking` needs a translation before the rotation, but it's different depending on the `IBaubleRender.RenderType`

**Comment by yrsegal** [comment]
Ok, let me rephrase.

[`translateToHeadLevel`](https://github.com/williewillus/Botania/blob/MC19/src/main/java/vazkii/botania/api/item/IBaubleRender.java#L48-L50) absolutely messes up when you sneak and change your ~~yaw~~ pitch.

[`rotateIfSneaking`](https://github.com/williewillus/Botania/blob/MC19/src/main/java/vazkii/botania/api/item/IBaubleRender.java#L39-L42) needs to call `GlStateManager.translate(0.0, 0.3, 0.0)` before the rotation is applied.

**Comment by yrsegal** [comment]
fold into #101
